sass/theme/_theme.scss

```diff
@@ -371,3 +371,18 @@ text-align: right;
 		}
 	}
 }
+
+
+/*--------------------------------------------------------------
+Dividers
+--------------------------------------------------------------*/
+.is-style-copy-divider{
+	@extend .copy-divider;
+}
+.block-editor-block-list__layout .wp-block.is-style-copy-divider{
+	margin: 0;
```

Comment:
I think this line might be the issue with how the gold divider looks in the editor. Setting all margins to 0 removes the built-in 48px top/bottom that the default divider has. Could we get away with only setting `margin-right` and `margin-left` here?